provision/aws/rds/aurora_common.sh

```diff
@@ -1,14 +1,10 @@
 #!/bin/bash
 set -e
 
-if [ -f ./.env ]; then
```

Comment:
I removed this because some env file may break removal if `AURORA_REGION` is set to some region, for exmaple, from some older deployment of single region aurora, then when removing global aurora env file rewrites value set by `aurora_delete_global_db.sh`. I think it is not necessary here. WDYT?

Reply:
That's fine by me. I only included this as it's the approach we have taken elsewhere.